.vsts-ci.yml

```diff
@@ -50,6 +50,35 @@ resources:
       tasks: true
 jobs:
 ###########################################
+- job: Maintenance
```

Comment:
I'm not sure exactly how Azure CI is configured but can't we run this independently of code pushes, e.g. once a day on a random runner?

Reply:
Fair enough, let's go with the straightforward solution then :) thanks for checking out the scheduled jobs!